<?xml version="1.0" ?>
<launch>


 <group ns="right_side">
	<arg name="right_hand_setting" default="$(find joint_carry)/config/right_qb_hand.yaml"/>
	<rosparam command="load" file="$(arg right_hand_setting)"/>
	<node name="qb_interface_node" pkg="qb_interface" type="qb_interface" args="0" output="screen" />
</group>


  <group ns="main_node">
    <param name="topic_name_right_robot_pose" value="/lwr/ee_pose"/>
    <param name="topic_name_left_robot_pose" value="/lwr/ee_pose"/>

    <param name="topic_name_right_hand_command" value="/right_side/qb_interface_node_/qb_class/hand_ref"/>
    <param name="topic_name_left_hand_command" value="/left_side/qb_interface_node_/qb_class/hand_ref"/>

    <param name="output_topic_name" value="/ds1/desired_velocity"/>
<!--     <rosparam command="load" file="$(find ds_motion_generator)/config/s4_back_GMM.yml"/>
 -->    <node name="controller_node" pkg="joint_carry" type="joint_carry_node" output="screen"/>

    <!-- # doesn't work yet, but I should follow this style -->
    <!-- <remap from="/ds1/filter/desired_velocity" to="/lwr/joint_controllers/passive_ds_command_vel" /> -->

  </group>





</launch>